docs/core/EigenPodManager.md

```diff
@@ -434,13 +434,16 @@ Whether each withdrawal is a full or partial withdrawal is determined by the val
 * Full withdrawals are performed when a Pod Owner decides to fully exit a validator from the beacon chain. To do this, the Pod Owner should follow these steps: 
     1. Undelegate or queue a withdrawal (via the `DelegationManager`: ["Undelegating and Withdrawing"](./DelegationManager.md#undelegating-and-withdrawing))
     2. Exit their validator from the beacon chain and provide a proof to this method
-    3. Complete their withdrawal (via [`DelegationManager.completeQueuedWithdrawal`](./DelegationManager.md#completequeuedwithdrawal))
+    3. Complete their withdrawal (via [`DelegationManager.completeQueuedWithdrawal`](./DelegationManager.md#completequeuedwithdrawal)).
+
+If the Pod Owner only completes steps 1 and 2 above, the ETH of the pod owner is still staked through EigenLayer and can be used to service AVSs, even though their ETH has been withdrawn from the beacon chain. The protocol allows for this edge case. 
```

Comment:
I realized this is basically about just completing step (2) -- step (1) doesn't matter for the edge case, right? and they could always "complete the withdrawal as shares" to keep these (re)staked ETH in EigenLayer

Reply:
Ah yes, good catch - updated.